src/main/java/com/icthh/xm/ms/dashboard/service/DashboardService.java

```diff
@@ -56,7 +55,6 @@ public Dashboard saveAndFlush(Dashboard dashboard) {
      */
     @Transactional(readOnly = true)
     @FindWithPermission("DASHBOARD.GET_LIST")
-    @PrivilegeDescription("Privilege to get all the dashboards")
```

Comment:
why did you delete @PrivilegeDescription?

Reply:
It's the result of a master merge. I restored it